### DISCUSSION TITLE
What is the status on the secondary and accent outline buttons?

### DISCUSSION THREAD

**Discussion by mworrest** [uswds/uswds#4451] [General]
Hi, my team would like to use the secondary outline button. What is the status on implementation of this? I found a pr on hold: https://github.com/uswds/uswds/pull/4257

Do we need to create our own variant or will you be implementing this anytime soon?

Thanks,
Team NCI

**Comment by mejiaj** [comment]
@mworrest thanks for your interest in this! One of the hold ups for this effort is ensuring that all combinations meet contrast requirements in **all** states.

Right now, it's probably best you use tokens to extend the button variants you need for your UI.
